.factorypath

```diff
@@ -0,0 +1,146 @@
+<factorypath>
```

Comment:
What is this file for?

Reply:
Looks like it's something that maven sometimes produces as a by-product of the build process-I certainly didn't ask for it.  Removing it from the build.

Reply:
Weird. Thanks for removing it.